src/components/Sponsors/Sponsors.css

```diff
@@ -0,0 +1,64 @@
+.sponsors-container {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  margin: 50px 0px;
+}
+
+.sponsors-title {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  margin-bottom: 20px;
+}
+
+.box {
+  display: flex;
+  justify-content: center;
+
+  left: 35px;
+  top: 4326px;
+  margin: auto;
+  margin-bottom: 20px;
+  background-color: #a57ed4;
+  color: #fff;
+  font-size: 44px;
+  text-align: center;
+}
+
+.button {
+  flex-wrap: wrap;
+  width: 556px;
+  left: 472px;
+  top: 4562px;
+
+  background: #ebebeb;
+  border-radius: 25px;
+  font-size: 35px;
+}
+.sponsors-row {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  margin: 20px;
+}
+
+img {
+  width: 275px;
+  margin-bottom: 7px;
+  background-size: cover;
+  border-radius: 0%;
+  margin: 20px;
+}
```

Comment:
Instead of applying this broadly (to an entire HTML element), see if you can change it to class names so it won't interfere with any other elements. I'm also not quite sure what `background-size: cover` is supposed to do - can you look over that?